docs/ashrae9012022/section22/Rule22-43.md

```diff
@@ -0,0 +1,40 @@
+# Chiller - Rule 22-43    
+**Schema Version:** 0.0.25   
+**Mandatory Rule:** False    
+**Rule ID:** 22-43      
+**Rule Description:** When using performance curves from Normative Appendix J, chiller minimum part-load ratio (ratio of load to available capacity at a given simulation time step) and minimum compressor unloading ratio (part-load ratio below which the chiller capacity cannot be reduced by unloading and chiller is false loaded) shall be equal to 0.25.       
+**Rule Assertion:** Baseline RMD = expected value   
+**Appendix G Section:** Section G3.2.2.1 Equipment Efficiencies      
+**90.1 Section Reference:** None  
+**Data Lookup:** None  
+**Manual Check:** No  
+**Evaluation Context:** Chiller  
+**Data Lookup:** None  
+**Function Call:** 
+1. is_chiller_performance_app_j()
+
+
+## Applicability Check:
+
+- **Check 1:** Get B-RMD system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMD)`
+
+- Check if B-RMD is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B"]): NEXT_APPLICABILITY_CHECK`
+
+- Else, rule is not applicable to B-RMD: `else: RULE_NOT_APPLICABLE`
+
+- Get primary and secondary loops for B-RMD: `primary_secondary_loop_dictionary = get_primary_secondary_loops_dict(B_RMD)`
+
+- For each chiller in B_RMD: `for chiller in B_RMD...chillers:`
+
+  - Check if chiller is connected to a primary chilled water loop that serves HVAC systems and has performance that aligns with App J: `if chiller.cooling_loop.id in primary_secondary_loop_dictionary and is_chiller_performance_app_j(chiller): CHECK_RULE_LOGIC`
```

Comment:
Just want to confirm it is intentional to use is_chiller_performance_app_j versus does_chiller_performance_match_curve